format/tmux/formater.go

```diff
@@ -33,6 +34,7 @@ type symbols struct {
 	Untracked string // Untracked is the string shown before the count of untracked files.
 	Stashed   string // Stashed is the string shown before the count of stash entries.
 	Clean     string // Clean is the string shown when the working tree is clean.
+	Delimiter string // Delimiter is the string shown before flags
```

Comment:
Since the order of components can now be re-ordered, I guess the Delimiter should represent the string that delimits them.